release-notes/VERSION-2.x

```diff
@@ -2585,7 +2585,7 @@ Other changes:
 
 * New variant of AnnotationIntrospector.getFormat(), to support class
   annotations
-* It is now possible to serialize instances of plain old Object, iff
+* It is now possible to serialize instances of plain old Object, if
```

Comment:
```suggestion
* It is now possible to serialize instances of plain old Object, if and only if
```

`iff (if and only if)` and `if` are --most probably-- two different things. So let's not diverge